src/web/calculators/linear/components/LinearCalculator.tsx

```diff
@@ -268,6 +325,50 @@ export default function LinearCalculator(): JSX.Element {
               defaultUnit="A"
             />
           </SingleInputLine>
+          <SingleInputLine
+            label="Estimated kG"
+            id="kG"
+            tooltip="Gravity feedforward constant of the mechanism (assumes vertical orientation)."
+          >
+            <MeasurementOutput
+              stateHook={[kG, setKg]}
+              numberRoundTo={2}
+              defaultUnit="V"
+            />
+          </SingleInputLine>
+          <SingleInputLine
+            label="Estimated kV"
+            id="kV"
+            tooltip="Velocity feedforward constant of the mechanism."
+          >
+            <MeasurementOutput
+              stateHook={[kV, setKv]}
+              numberRoundTo={2}
+              defaultUnit="V*s/m"
```

Comment:
there's no "kind" of unit for this in the library, so you'll have to add a new set of choices for units in `Measurement.choices`, in the same place that the `in^2 lbs` choice is defined.

Basically, that function takes a unit type and maps it to an array of valid, user-selectable unit choices for that unit type. However since these are uncommon unit types, I haven't defined what that array should be.

A quick fix would be to change the final return value of `Measurement.choices` from `[]` to `[m.units()]`, which yields this:
![image](https://user-images.githubusercontent.com/7595639/151265759-2fb856af-709f-41a3-a772-2c6c7e9528bd.png)

You may need to throw out a radian in kV. That can be quickly done in `myMeasurement.removeRad()`. The unit library doesn't handle radians very well.